src/test/regress/expected/multi_select_distinct.out

```diff
@@ -672,12 +671,13 @@ EXPLAIN (COSTS FALSE)
 		GROUP BY l_suppkey, l_linenumber
 		ORDER BY 1
 		LIMIT 10;
-                                                                 QUERY PLAN
+                                                                QUERY PLAN
 ---------------------------------------------------------------------
  Limit
-   ->  Unique
-         ->  Sort
-               Sort Key: (((pg_catalog.sum(remote_scan.dis))::bigint + COALESCE((pg_catalog.sum(remote_scan.dis_1))::bigint, '0'::bigint)))
+   ->  Sort
```

Comment:
is it ok that we're missing the Unique node here?

Reply:
I guess so. I am still not sure of the relevant PG commit. Could be this:
https://github.com/postgres/postgres/commit/db0d67db2401eb6238ccc04c6407a4fd4f985832#diff-fbea59d65b5b83952dd2522c9b2c2c16632c889eb402f4486e2cc39cdf0c47e2L6345
A bunch of `Unique` nodes are replaced by `HashAggregate` nodes. However, setting `enable_group_by_reordering` to `off` as suggested in the commit description didn't help.

In PG15 support PR, I have used the `coordinator_plan` function to avoid alternative output because of this HashAgg/HashJoin leverage 
https://github.com/citusdata/citus/pull/6085/commits/154e01141bb148790a244381946fe3c69b21ee9c
https://github.com/citusdata/citus/pull/6085/commits/2bdbea2503f48d5f0e8b5e03fa25baba40ad0bd6

Reply:
> Could be this:
https://github.com/postgres/postgres/commit/db0d67db2401eb6238ccc04c6407a4fd4f985832#diff-fbea59d65b5b83952dd2522c9b2c2c16632c889eb402f4486e2cc39cdf0c47e2L6345

It seems so

> However, setting enable_group_by_reordering to off

The value should be set on the workers. Did you maybe only set on the coordinator?

I think it is fine that we don't set it on the tests but can you locally run the tests where this value is off on the workers and check the EXPLAIN outputs are the same?

Reply:
> The value should be set on the workers. Did you maybe only set on the coordinator?

I have set in both the coordinator and the workers but still the explain output has HashAggregate.
This is what I used:
```SQL
ALTER SYSTEM SET enable_group_by_reordering TO off;
SELECT pg_reload_conf();
SELECT DISTINCT 1 FROM run_command_on_workers($$ALTER SYSTEM SET enable_group_by_reordering TO off;$$);
SELECT run_command_on_workers($$SELECT pg_reload_conf()$$);
```

Reply:
I looked a little more on the commit Naisila shared.

Ok, the mentioned GUC doesn't help, because that GUC is relevant for `Optimize order of GROUP BY keys`. However, some of the other costing related changes in that PR triggers these changes.

The changes all look sane to me, instead of `Unique` we have `Aggregate `. Similar changes exists on PG test changes as well

Reply:
Thanks for looking into it @onderkalaci